File: assets/samples/brand.alert/ruby/brand_alert_api_example.rb
require 'uri'
require 'json'
require 'net/https'

key = "Your Brand Alert 2.0 API key"
mode = "preview"
terms = ["facebook"]

url = 'https://brand-alert-api.whoisxmlapi.com/api/v2'
data = {apiKey: key, mode: mode, includeSearchTerms: terms}
uri = URI.parse(url)
http, http.use_ssl = Net::HTTP.new(uri.host, uri.port), true
req, req.body = Net::HTTP::Post.new(uri.request_uri), data.to_json
http.request(req).body

puts http.request(req).body

File: assets/samples/brand.alert/nodejs/brand_alert_api_example.js
const http = require('https');

const key = 'Your Brand Alert 2.0 API key';
const terms = ["facebook"];
const mode = "preview";
const host = 'brand-alert.whoisxmlapi.com';
const path = '/api/v2';
const data = {
    apiKey: key,
    includeSearchTerms: terms,
    mode: mode
};
const options = {
    method: "POST",
    hostname: host,
    path: path
};
var req = http.request(options, function(response) {
    var str = '';
    response.on('data', function(chunk) { str += chunk; });
    response.on('end', function() { console.log(str); });
});
req.write(JSON.stringify(data));
req.end();
